Models/ManagerDTO.cs

```diff
@@ -0,0 +1,99 @@
+﻿using AirportTickets.Models.Booking;
+using AirportTickets.Models.Flight;
+using AirportTickets.Models.Reservation;
+using static AirportTickets.Models.Flight.FlightDTO;
+
+namespace AirportTickets.Models.Manager
+{
+    public class ManagerDTO
+    {
+        public string Name { get; set; }
+
+        public ManagerDTO()
+        {
+
+        }
+
+        public List<ReservationDTO> FilterByFlight(FlightDTO flight)
```

Comment:
these methods should be moved out to a service class that handles the operations of reservations (ReservationService). This class is just a DTO, hold data, these methods here are irrelevant to the DTO.